app/controllers/relationships_controller.rb

```diff
@@ -0,0 +1,22 @@
+class RelationshipsController < ApplicationController
+  before_action :require_user
+
+  def index
+    @relationships = current_user.following_relationships
+  end
+
+  def create
+    leader = User.find(params[:leader_id])    
+    Relationship.create(leader_id: params[:leader_id], follower_id: current_user.id) if current_user.can_follow?(leader)
```

Comment:
This line is over 80 characters - to keep lines readable and to make debugging easier please refactor any lines over this limit. You can see this referenced in the style guide here - https://github.com/bbatsov/ruby-style-guide#80-character-limits